src/kind_lab_to_nwb/rat_behavioural_phenotyping_2025/requirements.txt

```
@@ -1,4 +1,5 @@
 ndx-events==0.2.1
+ndx-sound==0.2.1
```

Should be https://pypi.org/project/ndx-sound/0.2.2/
```suggestion
ndx-sound==0.2.2
```